packages/opensearch.opensearch.ephemeral/0.1.0-canary.0.277/.nupkg.metadata

```diff
@@ -0,0 +1,5 @@
+{
+  "version": 2,
+  "contentHash": "76uDTVvMlMqIw47JMlnMXAYoYjMocRgirue/XJP+YQ2p/UGtWD9m8OTvhwCKWHtEO5pZG8+YqAzjEiUYLbKZwg==",
+  "source": null
+}
```

Comment:
Does this need a new line?

Reply:
May be, but it is an autogenerated file.